Testscripts/Linux/docker_compose_wordpress_mysql_app.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+########################################################################
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+# Description:
+#   This script run wordpress application using docker compose.
+#
+########################################################################
+
+CONTAINER_NAME="wordpress_ex"
+
+function RunDockerCompose()
+{
+    mkdir wordpress
+    pushd wordpress
+    cat <<-EOF > "docker-compose.yml"
+version: '3.3'
```

Comment:
fixed version? any chance it will be changed? If that is the case in future, can you replace with the parameter?

Reply:
It will not change for this basic app.